### PR TITLE
Make scheduler_perf warn rather then fail if scheduling rate is betwe…

### PR DESCRIPTION
**Pull request by jayunit100**
**What this PR does / why we need it**

Fix scheduler per bench tests so they pass even if there is a cold interval .

**Special notes for your reviewer**:

This wont effect CI, as this bench test is run manually by devs.

### CODE REVIEW

File: test/integration/scheduler_perf/scheduler_test.go [hunk 1]
```diff
@@ -44,10 +45,13 @@ func TestSchedule100Node3KPods(t *testing.T) {
...
-	if min := schedulePods(config); min < threshold3K {
-		t.Errorf("Too small pod scheduling throughput for 3k pods. Expected %v got %v", threshold3K, min)
+	min := schedulePods(config)
+	if min < threshold3K {
+		t.Errorf("Failing: Scheduling rate was too low for an interval, we saw rate of %v, which is the allowed minimum of %v ! ", min, threshold3K)
+	} else if min < warning3K {
+		fmt.Printf("Warning: pod scheduling throughput for 3k pods was slow for an interval... Saw a interval with very low (%v) scheduling rate!", min)
 	} else {
-		fmt.Printf("Minimal observed throughput for 3k pod test: %v\n", min)
+		fmt.Printf("Minimal observed throughput for 3K pod test: %v\n", min)
```
Comment (gmarek) [inline test/integration/scheduler_perf/scheduler_test.go]: Why capital K?

**Comment by gmarek** [comment]
One nit + gofmt issues.

**Comment by jayunit100** [comment]
ping @gmarek ready to go on this one? thanks for the review !~

**Pull request merged**